static/503.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Code City</title>
+  <style>
+    body {
+      font-family: "Roboto Mono", monospace;
+      text-align: center;
+    }
+    h1 {
+      font-size: 40pt;
+      font-weight: 100;
+    }
+    h1>svg {
+      vertical-align: text-bottom;
+    }
+    #tagline {
+      font-style: italic;
+      margin: 2em;
+    }
+    iframe {
+      height: 50px;
+      width: 100px;
+      border: none;
+      display: block;
+      margin: 0 auto;
+    }
+  </style>
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet">
+  <link href="/static/favicon.ico" rel="shortcut icon">
+</head>
+<body>
+  <h1>
+    <svg xmlns="http://www.w3.org/2000/svg" width="95px" height="100px">
+      <path
+         d="m 1,90 0,-25 10,0 0,-8 9,0 0,-5 8,0 0,5 2,0 0,33 3,0 0,-13 2,0 0,-7 2,0 0,-32 1,0 0,-7 1,0 0,-6 2,0 0,-1 1,0 0,-12 1,0 0,-12 0,12 1,0 0,12 1,0 0,1 2,0 0,6 1,0 0,7 1,0 0,32 2,0 0,7 2,0 0,13 4,0 0,-32 5,0 0,-5 6,0 0,5 3,0 0,32 5,0 0,-42 3,0 0,-9 7,0 0,-2 5,0 0,2 2,0 0,51"
+         style="fill: none; stroke: #000; stroke-width: 1.5;" />
+    </svg>
+    Code City
+  </h1>
+  <p id="tagline">A community of inquisitive programmers.</p>
+  <p>Unfortunately this Code City is not available right now. 
+  Please come back later!</p>
```

Comment:
The target audience are programmers.  Can we create a less corporate-like message?  In this context it's ok to say we screwed up, it's ok to have fun, and it's ok to divulge any available technical details.

Reply:
Yes.  I've rewritten it to provide a bit more detail about possible causes (including incompetence on our part).  Do improve it further if you have other ideas.